### PR TITLE
Feature/planned assessment

### PR DESCRIPTION
**Pull request by AdityaKhatri**

### CODE REVIEW

File: src/views/EditAry/RightPanel/Focuses/index.js [hunk 1]
```diff
@@ -75,6 +75,8 @@ export default class Focuses extends React.PureComponent {
         const affectedGroupsTitle = _ts('editAssessment.methodology', 'affectedGroupsTitle');
         const locationsTitle = _ts('editAssessment.methodology', 'locationsTitle');
 
+        console.warn('here', geoOptions, projectDetails.regions);
```
Comment (tnagorra) [inline src/views/EditAry/RightPanel/Focuses/index.js]: Please sir! No sir!

File: src/views/Arys/FilterArysForm/index.js [hunk 2]
```diff
@@ -146,20 +136,33 @@ export default class FilterArysForm extends React.PureComponent {
...
 
     render() {
         const {
             className,
-            aryFilterOptions: {
-                createdBy,
+            requests: {
+                filterOptionsRequest: {
+                    response: {
+                        createdBy = [],
```
Comment (tnagorra) [inline src/views/Arys/FilterArysForm/index.js]: Empty list as default value for `createdBy` field?

File: src/views/EditAry/index.js [hunk 3]
```diff
@@ -149,6 +156,22 @@ const requestOptions = {
...
+        onFailure: ({
+            error,
+            props: {
+                activeLeadId,
+                activeLeadGroupId,
+                editAryServerId,
+                removeAry,
+            },
+        }) => {
+            if (error.response.errorCode === 404 && isDefined(editAryServerId)) {
```
Comment (tnagorra) [inline src/views/EditAry/index.js:170]: We may need to add some comment here!

File: src/views/PlannedArys/ActionButtons/index.js [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+                plannedAryDeleteRequest: {
+                    pending,
+                },
+            },
+            onPlannedAryEdit,
+        } = this.props;
+
+        return (
+            <div className={styles.actionButtons}>
+                {pending && <LoadingAnimation />}
```
Comment (tnagorra) [inline src/views/PlannedArys/ActionButtons/index.js:88]: Why not use Button's `pending` props

File: src/views/PlannedArys/PlannedAryForm/Group/index.js [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+const fieldKeySelector = data => data.id;
+
+export default class PlannedMetadataGroups extends React.PureComponent {
+    static propTypes = propTypes;
+    static defaultProps = defaultProps;
+
+    // FIXME: should identify using identifier
+    isStakeholderColumn = () => {
+        const { title } = this.props;
+        return title.toLowerCase() === 'stakeholders';
```
Comment (tnagorra) [inline src/views/PlannedArys/PlannedAryForm/Group/index.js:44]: We need to use ID mapping to identify "Stakeholder" Metadata Group

File: src/views/PlannedArys/index.js [hunk 6]
```diff
@@ -0,0 +1,421 @@
...
+            activePage,
+            activeProject,
+            requests: {
+                plannedArysGetRequest: {
+                    pending: loadingArys,
+                },
+            },
+        } = this.props;
+
+        if (redirectTo) {
```
Comment (tnagorra) [inline src/views/PlannedArys/index.js]: Is `redirectTo` ever set?

File: src/views/PlannedArys/ActionButtons/index.js (hunk 4 above)
Comment (AdityaKhatri) [reply src/views/PlannedArys/ActionButtons/index.js:88]: For dramatic effects, we use loading animation

**Review by tnagorra — CHANGES_REQUESTED** [review]
Need to fix some changes!

**Pull request merged**